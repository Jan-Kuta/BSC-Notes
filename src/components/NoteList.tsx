import React from 'react';
import {NavLink} from 'react-router-dom';

export const NoteList = () => (
    <>
        <NavLink to="/1">Detail</NavLink>
        <h1>NOTE LIST</h1>
    </>
);
