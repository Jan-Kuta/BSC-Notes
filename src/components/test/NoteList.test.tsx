import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';

import {NoteList} from '../NoteList';

it('NoteList renders correctly', () => {
    const tree = renderer
        .create(
            <MemoryRouter>
                <NoteList />
            </MemoryRouter>
        )
            .toJSON();
    expect(tree).toMatchSnapshot();
});
