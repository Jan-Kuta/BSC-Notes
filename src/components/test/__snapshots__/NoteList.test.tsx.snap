// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteList renders correctly 1`] = `
Array [
  <a
    aria-current={null}
    href="/1"
    onClick={[Function]}
  >
    Detail
  </a>,
  <h1>
    NOTE LIST
  </h1>,
]
`;
